### PR TITLE
feature/#4429/Add_a_service_Послуга_to_the_tariffs_card_#4429

### PR DESCRIPTION
**Pull request by SokhaTaras**
In this PR,I had made some methods,which allow us to make request to back-end in order to make changes in cards

### CODE REVIEW

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-location-dashboard.component.ts [hunk 1]
```diff
@@ -389,8 +389,8 @@ export class UbsAdminTariffsLocationDashboardComponent implements OnInit, OnDest
     return cityArray;
   }
 
-  page(locationID): void {
-    this.router.navigate([`ubs-admin/tariffs/location/${locationID}`]);
+  page(cardID): void {
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-location-dashboard.component.ts:392]: cardID: number

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-pricing-page.component.ts [hunk 2]
```diff
@@ -81,54 +81,115 @@ export class UbsAdminTariffsPricingPageComponent implements OnInit, OnDestroy {
...
-      maxAmountOfBigBag,
-      maxAmountOfOrder,
+  sumToggler() {
+    this.toggle = true;
+  }
+
+  bagToggler() {
+    this.toggle = false;
+  }
+
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-pricing-page.component.ts:113]: These functions (sumToggler and bagToggler) are redundant

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-pricing-page.component.ts [hunk 3]
```diff
@@ -81,54 +81,115 @@ export class UbsAdminTariffsPricingPageComponent implements OnInit, OnDestroy {
...
+    if (this.toggle === null) {
+      this.tariffsService
+        .setLimitDescription(this.descriptionInfo.limitDescription, courierId)
+        .pipe(takeUntil(this.destroy))
+        .subscribe(() => {
+          this.getCouriers();
+        });
+    }
+
+    if (this.toggle === true) {
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-pricing-page.component.ts]: if (this.toggle)

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-pricing-page.component.ts [hunk 4]
```diff
@@ -81,54 +81,115 @@ export class UbsAdminTariffsPricingPageComponent implements OnInit, OnDestroy {
...
-      maxAmountOfOrder,
+  sumToggler() {
+    this.toggle = true;
+  }
+
+  bagToggler() {
+    this.toggle = false;
+  }
+
+  async saveChanges() {
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-pricing-page.component.ts:114]: add return type - Promise<void>

Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-pricing-page.component.ts]: add return type - Promise < void >

File: src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-pricing-page.component.ts [hunk 5]
```diff
@@ -81,54 +81,115 @@ export class UbsAdminTariffsPricingPageComponent implements OnInit, OnDestroy {
...
+    } catch (e) {
+      return Error('getCourierId Error');
+    }
   }
 
   routeParams(): void {
     this.route.params.pipe(takeUntil(this.destroy)).subscribe((res) => {
       this.getAllTariffsForService();
-      this.selectedLocationId = +res.id;
+      this.selectedCardId = +res.id;
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-pricing-page.component.ts]: Number(res.id)

**Review by hnativlyubomyr — APPROVED** [review]

**Review by atsa21 — APPROVED** [review]

**Review by zaitsev-oleksii — APPROVED** [review]

**Review by katia9261 — APPROVED** [review]

**Review by vplvua — APPROVED** [review]

**Review by marianaharnyk — APPROVED** [review]

**Review by KhristinaVasko — APPROVED** [review]

**Pull request merged**